src/sap.ui.table/src/sap/ui/table/TableDragAndDropExtension.js

```diff
@@ -4,11 +4,12 @@
 
 // Provides helper sap.ui.table.TableDragAndDropExtension.
 sap.ui.define([
-	"./TableExtension", "sap/ui/table/TableUtils", "sap/ui/core/dnd/DropPosition"
-], function(TableExtension, TableUtils, DropPosition) {
+	"./TableExtension", "sap/ui/table/TableUtils", "sap/ui/core/library"
+], function(TableExtension, TableUtils, library) {
```

Comment:
Suggestion: better name it 'coreLibrary' to avoid confusion wit the tableLibrary.